x-pack/legacy/plugins/apm/common/elasticsearch_fieldnames.test.ts

```diff
@@ -13,7 +13,7 @@ import * as fieldnames from './elasticsearch_fieldnames';
 
 describe('Transaction', () => {
   const transaction: AllowUnknownProperties<Transaction> = {
-    '@timestamp': new Date().toString(),
+    '@timestamp': new Date(1337).toString(),
```

Comment:
nit: I don't think it matters much, but `@timestamp` is ISO 8601 format. We can get the same with `new Date(1337).toUTCString()`